nasa-gibs/worldview
# wv-2830: Crash fix: HLS_True_Color_Landsat continued cleanup

## Description
A bug was recently introduced which leads to a crash if a user tries to add "Featured" layers. This bug is present on the current develop branch & can be triggered by selecting Add Layers & then selecting the Featured tab. This fix surfaced a few other instances requiring removal of layers as well.

Fixes # wv-2830

## How To Test

- git checkout wv-2830-features-crash
- npm ci
- npm run build
- npm run watch
- Click on Add Layers
- Click on the "Featured" tab
- Observe that the app no longer crashes

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
